src/containers/Tenant/Diagnostics/Overview/TableInfo/prepareTableInfo.ts

```diff
@@ -1,14 +1,148 @@
-import type {TEvDescribeSchemeResult} from '../../../../../types/api/schema';
-
-import {formatObject} from '../../../../../components/InfoViewer';
+import type {
+    TColumnDataLifeCycle,
+    TColumnTableDescription,
+    TEvDescribeSchemeResult,
+    TPartitionConfig,
+    TTTLSettings,
+} from '../../../../../types/api/schema';
+import type {KeyValueRow} from '../../../../../types/api/query';
+import {EPathType} from '../../../../../types/api/schema';
+import {isNumeric} from '../../../../../utils/utils';
+import {formatBytes, formatNumber} from '../../../../../utils';
+import {formatDurationToShortTimeFormat} from '../../../../../utils/timeParsers';
+import {formatObject, InfoViewerItem} from '../../../../../components/InfoViewer';
 import {
     formatFollowerGroupItem,
     formatPartitionConfigItem,
     formatTableStatsItem,
     formatTabletMetricsItem,
 } from '../../../../../components/InfoViewer/formatters';
 
-export const prepareTableInfo = (data?: TEvDescribeSchemeResult) => {
+const isInStoreColumnTable = (table: TColumnTableDescription) => {
+    return 'SchemaPresetName' in table && 'SchemaPresetId' in table;
+};
+
+const prepareOlapStats = (olapStats?: KeyValueRow[]) => {
+    const Bytes = olapStats?.reduce((acc, el) => {
+        const value = isNumeric(el.Bytes) ? Number(el.Bytes) : 0;
+        return acc + value;
+    }, 0);
+    const Rows = olapStats?.reduce((acc, el) => {
+        const value = isNumeric(el.Rows) ? Number(el.Rows) : 0;
+        return acc + value;
+    }, 0);
+    const tabletIds = olapStats?.reduce((acc, el) => {
+        acc.add(el.TabletId);
+        return acc;
+    }, new Set());
+
+    return [
+        {label: 'PartCount', value: tabletIds?.size ?? 0},
+        {label: 'RowCount', value: formatNumber(Rows) ?? 0},
+        {label: 'DataSize', value: formatBytes(Bytes) ?? 0},
```

Comment:
Labels and values format was changed to be the same with labels and values from `TableStats`